orderbook/openapi.yml

```diff
@@ -417,6 +417,49 @@ paths:
                   $ref: "#/components/schemas/Order"
         400:
           description: Problem with parameters like limit being too large.
+  /api/v1/quote:
+    post:
+      summary: Quotes a price and fee for the specified order parameters.
+      description: |
+        This API endpoint accepts a partial order and computes the minimum fee and
+        a price estimate for the order. It returns a full order that can be used
+        directly for signing, and with an included signature, passed directly to
+        the order creation endpoint.
+      parameters:
+        - in: header
+          name: X-AppData
```

Comment:
This is to avoid the IPFS hash lookup when receiving e.g. a CowSwap app data that also contains a referrer (making the AppData hash unique), right? 
I think that we have to resolve the IPFS hash anyways when verifying the order's fee upon placement and given this can be easily cached, I'm actually no longer sure we should provide the raw interface ID as an extra param.

Reply:
Sounds good. We can also add this back in if we run into technical issues with IPFS. Lets leave it out for now.

Reply:
Why X prefix?

that naming convention  is usually used for header params, but not for the normal params, right?

Reply:
This was a header param. We have since removed it.